packages/fxa-auth-client/lib/client.ts

```diff
@@ -71,6 +71,12 @@ export type SignedInAccountData = {
   unwrapBKey?: hexstring;
 };
 
+export type SessionReauthOptions = SignInOptions;
```

Comment:
I noticed that the SignInOpitons type has an optional unblock code, should this be omitted? Since it's optional, maybe it doesn't matter?

Reply:
The reauth endpoint also has it: https://github.com/mozilla/fxa/blob/d1d00db33f1d7073099ea248ed491977e7645aac/packages/fxa-auth-server/lib/routes/session.js#L123  Unless you meant something else?